lib/api/apiUtils/authorization/prepareRequestContexts.js

```diff
@@ -1,12 +1,18 @@
 const { policies } = require('arsenal');
 const { config } = require('../../../Config');
 
-const RequestContext = policies.RequestContext;
-const requestUtils = policies.requestUtils;
+const { RequestContext, requestUtils } = policies;
 let apiMethodAfterVersionCheck;
-const apiMethodWithVersion = { objectGetACL: true, objectPutACL: true,
-    objectGet: true, objectDelete: true, objectPutTagging: true,
-    objectGetTagging: true, objectDeleteTagging: true };
+const apiMethodWithVersion = {
```

Comment:
Probably out of the scope of your work, but using a Set here would probably be more natural than an object where all values are ignored.

Reply:
Yeah totally agreed. That would require a bit of refactoring tho, so not gonna include it in this pr, but I've got a running list of cloudserver refactors that I'll add it to :')